test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Uses/dataset.toml

```diff
@@ -1,13 +1,13 @@
 data_source = "AEO2021 Reference Case"
 dataset_id = "aeo2021_ref_com_energy_end_uses"
 data_schema_type = "one_table"
-dataset_type = "benchmark" # <--- ?
-path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Uses/data/" # <---
+dataset_type = "benchmark"
+path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Uses/data/"
 description = "AEO 2021 Reference Case Commercial Energy End Uses Projection: 2013-2050"
 origin_creator = "Lixi Liu"
 origin_organization = "NREL"
 origin_contributors = [ "Lixi Liu", "Meghan Mooney", "Elaine Hale",]
-origin_project = "Energy Benchmarking and Projection" # <---
+origin_project = "Energy Benchmarking and Projection"
```

Comment:
Since these data are a straight translation of the detailed AEO data, I'd say something like

```
origin_creator = "EIA"
origin_organization = "EIA"
origin_contributors = ["EIA"]
origin_project = "U.S. Annual Energy Outlook 2021"
```

It's probably safe to say that Kevin Jarzomski created these data, but we don't generally have that kind of information for external data.

Reply:
Alternatively we can add a new field called `data_collector` to specify for that traceability.